modules/globebrowsing/src/timequantizer.cpp

```diff
@@ -366,13 +366,17 @@ void TimeQuantizer::setStartEndRange(const std::string& start, const std::string
 
 void TimeQuantizer::setResolution(const std::string& resolutionString) {
     _resolution = parseTimeResolutionStr(resolutionString);
+    verifyStartTimeRestrictions();
 }
 
 void TimeQuantizer::verifyStartTimeRestrictions() {
-    if (_start.day() < 1 || _start.day() > 28) {
+    //If monthly time resolution then restrict to 28 days so every month is consistent
+    unsigned int dayUpperLimit = (_resolutionUnit == 'M') ? 28 : 31;
```

Comment:
I'd also check for yearly;  otherwise someone might want to ask for a start date of february 29th in a leap year and we have same issue.
Super edge case, but still

Reply:
Added a case for yearly in commit dd6aba5